public_html/wp-content/plugins/camptix/inc/class-camptix-badges.php

```diff
@@ -48,7 +48,8 @@ function process_badges() {
  * Outputs manage badge admin screen, and allows processing of submit.
  */
 function menu_badges() {
-	if ( isset( $_GET['badge-submit'] ) && ( 1 == $_GET['badge-submit'] ) ) {
+	// phpcs:ignore WordPress.Security.NonceVerification.Missing
+	if ( ! empty( $_POST ) ) {
```

Comment:
I wonder if we should check something more specific here, like `$_POST['action']`?

Reply:
I considered it, but then figured who cares :) Either the handler will pass or it won't.. 

Appreciate you looking and merging @pkevan!